webpack.config.js

```diff
@@ -5,7 +5,8 @@ module.exports = {
   mode: isDev ? 'development' : 'production',
 
   entry: {
-    "HelpLightningClient": ['./src/core/HelpLightningClient.js']
+    "HelpLightningClient": ['./src/core/HelpLightningClient.js'],
+    "invite": ['./src/core/features/invite/components/Invite']
```

Comment:
I think we should make this:
```
    "components/Invite": ['./src/core/features/invite/components/Invite']
```
Then you'd do something like:
```
import {Invite} from 'components/Invite.js'
```
right?

Reply:
There is a way to export the modules that user can import like below.
```
import { HelpLightningClient, components } from 'helplightning-web-components'; // eslint-disable-line
const { Invite } = components;
```
Is this OK?

Reply:
By our current export approach, user will import components like,
`import Invite from 'helplightning-web-components/dist/components/Invite'`.

Reply:
ok, I prefer the current `import Invite from 'helplightning-web-components/dist/components/Invite`. It'd be great if we eventually submitted this to npmjs. Do you know how we'd get rid of the dist? Is it just changing how we pack it?